cmd/gapit/report.go

```diff
@@ -34,9 +34,7 @@ type reportVerb struct{ ReportFlags }
 func init() {
 	verb := &reportVerb{
 		ReportFlags: ReportFlags{
-			CommandFilterFlags: CommandFilterFlags{
-				Context: -1,
-			},
+			CommandFilterFlags: CommandFilterFlags{},
 		},
```

Comment:
Get rid of all this?

Reply:
I want to reuse this machinery for more powerful command filtering soon. Just removing the dead filter criteria

Reply:
i don't think we want to filter the report, though, so here we should remove it.